assets/src/dashboard/app/views/editorSettings/index.js

```diff
@@ -338,6 +375,26 @@ function EditorSettings() {
               onCheckboxSelected={toggleWebStoriesTrackingOptIn}
               selected={optedIn}
             />
+            {canManageSettings && (
+              <AdNetworkSettings
+                handleUpdate={handleUpdateAdLoader}
+                adNetwork={adNetwork}
+              />
+            )}
+            {canManageSettings && 'adsense' === adNetwork && (
+              <GoogleAdSenseSettings
+                handleUpdatePublisherId={handleUpdateAdSensePublisherId}
+                handleUpdateSlotId={handleUpdateAdSenseSlotId}
+                publisherId={adSensePublisherId}
+                slotId={adSenseSlotId}
+              />
+            )}
+            {canManageSettings && 'admanager' === adNetwork && (
+              <GoogleAdManagerSettings
+                handleUpdate={handleUpdateAdManagerSlotId}
+                slotId={adManagerSlotId}
+              />
+            )}
```

Comment:
Ignore the bad formatting, but I think only one `canManageSettings` check would improve readability
```js
            {canManageSettings && (
              <>
	                <AdNetworkSettings
	                  handleUpdate={handleUpdateAdLoader}
	                  adNetwork={adNetwork}
	                />

	              {'adsense' === adNetwork && (
	                <GoogleAdSenseSettings
	                  handleUpdatePublisherId={handleUpdateAdSensePublisherId}
	                  handleUpdateSlotId={handleUpdateAdSenseSlotId}
	                  publisherId={adSensePublisherId}
	                  slotId={adSenseSlotId}
	                />
	              )}
	              {'admanager' === adNetwork && (
	                <GoogleAdManagerSettings
	                  handleUpdate={handleUpdateAdManagerSlotId}
	                  slotId={adManagerSlotId}
	                />
	              )}
			</>
           )}
```